### DISCUSSION TITLE
How do I link to specific "pages" in Hexo, not posts?

### DISCUSSION THREAD

**Discussion by 825i** [hexojs/hexo#5241] [Question / Answer]
Big thanks to making Hexo, I already love it.

I'm trying to make my blog here. [placeholder](https://blog.xtu.icu/)

What I would like is at the top, where "About" is, to link to a specific **page** on my blog. That is, a page which is separate from the way **posts** are displayed. Similar to in Wordpress when you make a page, it is treated independently to a post. Please excuse me if my question is confusing.

I have my installation directory as below:

![Screenshot 2023-07-13 155546](https://github.com/hexojs/hexo/assets/58630825/d83baf62-b811-4a6b-b768-2f89854c08c6)

According to the Docs: https://hexo.io/docs/writing
I should be able to put new pages in `root/source` but then my question is, how do I link them? i.e. how do I make the "About" link at the top, point to the `root/source/about.md`?

Thanks greatly for any help!

**Comment by stevenjoezhang** [comment]
You can try:
```
[About](/about.html)
```

**Comment by 825i** [comment]
Thanks for the reply! I found my answer here. https://github.com/EvanNotFound/hexo-theme-redefine/issues/182
